CHANGELOG.md

```diff
@@ -7,9 +7,9 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Next release
 
-## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.1.0...HEAD)] - YYYY-MM-DD]
```

Comment:
Please leave this line unchange

Reply:
Ahh it looks like @barryib in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/714

Reply:
I removed that from my changelog.md and put it back to match master.